src/app/item-popup/SocketDetails.tsx

```diff
@@ -77,7 +78,17 @@ function mapStateToProps() {
       const plugAllowList = new Set(socketType.plugWhitelist.map((e) => e.categoryHash));
       for (const item of allItems) {
         const itemDef = defs!.InventoryItem.get(item.hash);
-        if (itemDef.plug && plugAllowList.has(itemDef.plug.plugCategoryHash)) {
+        if (
+          itemDef.plug &&
+          plugAllowList.has(itemDef.plug.plugCategoryHash) &&
+          // Filter out items that reside in armor buckets (e.g. regular
+          // instanced armor whose inventory item def double-functions
+          // as a universal ornament plug). This might not be the proper
+          // way to do this. Maybe check API responses after Companion App
+          // gets universal ornaments right?
+          (!itemDef.inventory ||
+            !Object.values(armorBuckets).includes(itemDef.inventory?.bucketTypeHash))
```

Comment:
@bhollis Since we don't have PlugSets here, I tried to find a way to exclude these items based on the item def alone. This could also be a way to avoid hardcoding specific hashes to identify ornament plugs.

Reply:
What we should maybe do is only look at items which are in the mods bucket:

```
item.location.hash === BucketHashes.Modifications
```

That's the best I can think of to filter to the intention of InventorySourced plugs.

Reply:
I could live with that, but it that would change the behavior for mods in the vault (or postmaster). The companion app shows mods that are in the vault as available, but will fail to apply them.